make_a_release.sh

```diff
@@ -75,8 +75,8 @@ format_changelog() {
 new_ver="$1"; shift
 
 # Validate the new version
-printf "%s" "$new_ver" | egrep -q -x '[[:digit:]]+\.[[:digit:]]+' ||
-        fatal "please, provide new version in X.Y format"
+printf "%s" "$new_ver" | egrep -q -x '[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+' ||
+        fatal "please, provide new version in X.Y.Z format"
```

Comment:
Very happy to see this. I dislike the restrictions of "two digit" versions.

Reply:
I'm glad you agree! I want to be able to push out ".Z" releases for bug fixes instead of just "X.Y" releases.

I thought it curious that the comments say something about this code following semantic versioning 2.0.0, which mandates x.y.z versioning, but then didn't.